llvm/lib/Passes/PassRegistry.def

```diff
@@ -450,6 +452,7 @@ LOOP_PASS("loop-reduce", LoopStrengthReducePass())
 LOOP_PASS("indvars", IndVarSimplifyPass())
 LOOP_PASS("loop-unroll-full", LoopFullUnrollPass())
 LOOP_PASS("print-access-info", LoopAccessInfoPrinterPass(dbgs()))
+LOOP_ANALYSIS("oksana-loop-analysis", OksanaKozlovaLoopAnalysis())
```

Comment:
@ksana-kozlova, please move this to `LOOP_ANALYSIS` ruction of the file started at line 427